examples/min_vertex.rst

```diff
@@ -64,7 +64,8 @@ Solve the Problem by Sampling
 =============================
 
 For small numbers of variables, even your computer's CPU can solve minimum vertex covers
-quickly. Here we solve both classically on your CPU and on the quantum computer.
+quickly. In this example, we will demonstrate how to solve the problem both classically 
```

Comment:
Present tense: "In this example, we demonstrate how"

Reply:
Thank you.